### DISCUSSION TITLE
Copilot offers php solutions instead of HTML.

### DISCUSSION THREAD

**Discussion by NathanIJpma** [community/community#18963] [Copilot Conversations]
When I ask the copilot to create a table with 10 by 10 rows, it only offers php solutions, while I need a html solution.
![image](https://user-images.githubusercontent.com/98284258/174552158-075867bf-346a-4a07-9bc2-4c54db01f19b.png)

**Comment by kirollosatef** [comment]


**Reply by NathanIJpma** [reply]
Read the code :)
